StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/CustomerSheet/CustomerSheetConfiguration.swift

```diff
@@ -86,6 +86,11 @@ extension CustomerSheet {
         /// If false (default), only card brand choice eligible cards can be edited and users can remove payment methods from the list screen.
         @_spi(AlternateUpdatePaymentMethodNavigation) public var alternateUpdatePaymentMethodNavigation = false
 
+        /// This is an experimental feature that may be removed at any time.
+        /// If true, users can set a payment method as default.
```

Comment:
We should add that this also influences where we read the default payment method from, not just allow setting.

Reply:
What about "users can set a payment method as default and sync their default payment method across web and mobile"